packages/phone-number-privacy/test/index.test.ts

```diff
@@ -1,76 +1,124 @@
-import { BLINDBLS } from 'bls12377js-blind'
 import { getSalt } from '../src/index'
+import { computeBLSSalt } from '../src/salt-generation/bls-salt'
+import { QueryQuota } from '../src/salt-generation/query-quota'
 
+const BLS_SALT = '6546544323114343'
+const queryQuota: QueryQuota = new QueryQuota()
+
+jest.mock('../src/common/identity', () => ({
+  authenticateUser: jest.fn().mockReturnValue(true),
+}))
+
+jest.mock('../src/salt-generation/query-quota')
+const mockReaminingQueryCount = queryQuota.getRemainingQueryCount as jest.Mock
+mockReaminingQueryCount.mockResolvedValue(2)
+
+jest.mock('../src/salt-generation/bls-salt')
+const mockBlsSalt = computeBLSSalt as jest.Mock
+mockBlsSalt.mockReturnValue(BLS_SALT)
+
+jest.mock('../src/database/wrappers/account', () => ({
+  incrementQueryCount: jest.fn().mockReturnValue(new Promise((resolve) => resolve())),
```

Comment:
Nit a more common form for this would be  would be 
`jest.fn( () => {new Promise((resolve) => resolve())}),`

Also why the promise here? Keep in mind `await` can used on non-promises, Js handles it fine :)

Reply:
RE: promises -- i didn't find that to always be the case -- if I had a .catch() after the promise call, then jest would complain that just passing the value back didn't have a .catch() method